.eslintignore

```diff
@@ -1,2 +1,3 @@
 build
-example/contracts
\ No newline at end of file
+example/contracts
+*.test.*
```

Comment:
maybe ignore the specific rule that's bothering in eslintConfig in package?

Reply:
i dont think linting test files would give people much benefit though